generators/createDisplayUnitSet/templates/shared/css/style.css

```diff
@@ -60,10 +57,10 @@ body {
   left: 50%;
   width: 80%;
   height: 50%;
+  font-family: OpenSans, sans-serif;
   font-size: 50px;
   transform: translate(-50%, -50%);
   text-align: center;
-  text-transform: uppercase;
 }
 
 .title {
```

Comment:
if you rename it .text, which I find it a quite common name, this class should be .text as well :). Maybe rename title to copy?

Reply:
Actually, we've found that you can run into issues in the framework when the variable name is "text" or "copy" so a descriptive variable name was chosen here